src/main/java/org/javawebstack/abstractdata/mapper/annotation/MapperOptions.java

```diff
@@ -0,0 +1,21 @@
+package org.javawebstack.abstractdata.mapper.annotation;
+
+import org.javawebstack.abstractdata.mapper.MapperTypeAdapter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface MapperOptions {
+
+    String name() default "";
+    int order() default 0;
+    Class<?>[] generic() default {};
```

Comment:
Is the generic needed here? How reliable is the Field#getGenericType() method?

Reply:
Yes, unfortunately it is. Whether debug information is included or not is highly compiler and runtime dependent and not reliable at all.